### PR TITLE
Issue 4513 - CI Tests - fix test failures

### PR DESCRIPTION
**Pull request by mreynolds389**
Description:

    Fixed tests in these suites:  basic, entryuuid, filter, lib389, and schema

relates: https://github.com/389ds/389-ds-base/issues/4513

### CODE REVIEW

**Review by progier389 — APPROVED** [review]
LGTM

File: dirsrvtests/tests/suites/entryuuid/basic_test.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def _entryuuid_import_and_search(topology):
     target_ldif = os.path.join(ldif_dir, 'localhost-userRoot-2020_03_30_13_14_47.ldif')
     import_ldif = os.path.join(DATADIR1, 'localhost-userRoot-2020_03_30_13_14_47.ldif')
     shutil.copyfile(import_ldif, target_ldif)
+    os.chmod(target_ldif, 0o777)
```
Comment (Firstyear) [inline dirsrvtests/tests/suites/entryuuid/basic_test.py:40]: Should be 0o666.

Comment (mreynolds389) [reply dirsrvtests/tests/suites/entryuuid/basic_test.py:40]: Does that really matter for a CI test?

Comment (Firstyear) [reply dirsrvtests/tests/suites/entryuuid/basic_test.py:40]: It's not the end of the world, but also we should do it right IMO. :) it's not a deal breaker.

**Pull request merged**